website/README.md

```diff
@@ -14,19 +14,26 @@ Use the provided _requirements.txt_ file to setup your python environment.
 
 # Running a debug setup
 
-**From the _front_end_ folder**
+**From the _website_ folder**
+
+Create a python virtual env with the required libraries :
+```
+python3.7 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
 
 Create the scratch folder where the debug db will be stored and optionally create dummy data:
 ```
-$> mkdir storage
-$> python3 python/make_dummy_data.py
+mkdir storage
+python python/make_dummy_data.py
```

Comment:
What's wrong with python3? If the default is not python3 on the system it's going to be weird...

Reply:
I updated the readme to create a venv from python 3.7, so by calling "python" we make sure we rely on the venv.